pb/metrics.proto

```diff
@@ -34,10 +33,18 @@ message PodNetworkMetric {
   int64 rx_bytes = 13;
   int64 rx_packets = 14;
   int32 proto = 15;
-  string src_dns_name = 16;
-  string dst_dns_name = 17;
+  string dst_dns_name = 16;
```

Comment:
This breaks contract. You should leave this as 17, and add `reserved 16, "src_dns_name";`.